crates/sui-types/src/crypto.rs

```diff
@@ -1318,29 +1318,29 @@ impl<const STRONG_THRESHOLD: bool> AuthoritySignInfoTrait
 
 impl<const STRONG_THRESHOLD: bool> AuthorityQuorumSignInfo<STRONG_THRESHOLD> {
     pub fn new_from_auth_sign_infos(
```

Comment:
One thing that is strange about this function is that it trusts the caller to verify that all sign_infos came from different authorities, but not that the quorum threshold was reached. Not sure we want the second assumption, wdyt?